birdhouse/config/thredds/default.env

```diff
@@ -6,6 +6,15 @@ export THREDDS_DOCKER=pavics/thredds-docker
 export THREDDS_IMAGE="${THREDDS_DOCKER}:${THREDDS_VERSION}"
 export THREDDS_ORGANIZATION="Birdhouse"
 
+# All default values for data locations are set to ensure backward compatibility
+export STAC_DATA_LOCATION_FOR_THREDDS_ON_CONTAINER="/pavics-ncml"
```

Comment:
`CATALOG_` might bring confusion with `catalog` service (although deprecated, other similar variables can still be defined).
If the intent is to have a generic "dataset" shared between many services, `BIRDHOUSE_DATA_...` or simply `DATA_...` seems reasonable.

Reply:
I would like to propose to rename `MISC_DATA_LOCATION_NAME` ==> `RAW_DATA_LOCATION_NAME`, `MISC_DATA_LOCATION_FOR_THREDDS_ON_HOST` ==> `RAW_DATA_LOCATION_FOR_THREDDS_ON_HOST` and `MISC_DATA_LOCATION_FOR_THREDDS_ON_CONTAINER` ==> `RAW_DATA_LOCATION_FOR_THREDDS_ON_CONTAINER`.

`STAC_DATA_LOCATION_NAME` ==> `DATA_LOCATION_NAME` and similar for all its friends.

Avoid using `STAC` because this is the Thredds component, it should not know which other component(s) is/are crawling its data.

Reply:
@tlvu thank you for the naming suggestions. I agree that those are all good names and that I'm happy to use those names.